packages/preview/lacy-ubc-math-project/0.2.0/README.md

```diff
@@ -0,0 +1,24 @@
+# Lacy-UBC Math Project Template
+This is a Typst template for UBC Math 100/101 group projects. It aims to reproduce most of the existing LaTex template while providing a more user-friendly interface for students who are not familiar with LaTex.
+
+For more information, please refer to the [documentation](/manual.pdf).
```

Comment:
The relative link will also work when the file is not placed at the root of the repository, e.g. here in the package repository.
```suggestion
For more information, please refer to the [documentation](manual.pdf).
```